src/components/CommentsList.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react"
+import { CommentsListItem } from "./CommentsListItem"
+import { getCommentsByArticleId } from "../utils/api"
+
+export const CommentsList = ({article_id}) => {
+
+    const [comments, setComments] = useState([])
```

Comment:
Please create a loading state, and display a loading message to the user while the comments are being loaded.